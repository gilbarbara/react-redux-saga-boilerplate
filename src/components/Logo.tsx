import SVG from 'react-inlinesvg';
import styled from '@emotion/styled';

export const Wrapper = styled.div`
  align-items: flex-start;
  display: inline-flex;
  font-size: 0;

  svg {
    height: 42px;
    max-height: 100%;
    width: auto;
  }
`;

function Logo() {
  return (
    <Wrapper>
      <SVG src="/media/brand/icon.svg" />
    </Wrapper>
  );
}

export default Logo;
