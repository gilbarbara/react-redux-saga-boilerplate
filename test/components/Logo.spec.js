import React from 'react';
import { mount } from 'enzyme';
import Logo from 'components/Logo';

describe('Logo', () => {
  const wrapper = mount(<Logo />);

  it('should be a StatelessComponent', () => {
    expect(wrapper.instance()).toBeNull();
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
