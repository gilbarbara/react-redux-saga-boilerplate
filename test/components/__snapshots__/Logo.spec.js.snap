// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render properly 1`] = `"<div class=\\"app__logo\\"><span class=\\"isvg failed\\"><img src=\\"\\" alt=\\"react-redux-saga-boilerplate\\"></span></div>"`;
